Procfile

```diff
@@ -1 +1,2 @@
+release: bundle exec rake db:migrate
```

Comment:
The only time it doesn't make sense to me is a rollback. Wouldn't you need to call `rake db:rollback` accordingly then? Given that a migration was added in that code, you'd want to change that back as well, wouldn't you? Or does rollback refer to a step back in the pipeline (like from dev to staging)? And if that is the case, in this example is this then called on the dev or the staging pipeline step?